src/components/product/ProductCard/ProductCard.tsx

```diff
@@ -16,14 +16,23 @@ import {
   CardActions as UICardActions,
 } from '@faststore/ui'
 
-import './product-summary.scss'
+import './product-card.scss'
+
+type Variant = 'horizontal' | 'vertical'
 
 interface Props {
   product: ProductSummary_ProductFragment
   index: number
+  variant?: Variant
```

Comment:
what do you think about using `orientation` instead `variant`, wouldn't be better? Not a blocker at all, just thinking